website/docs/d/policy_ods_pre_defined_runbook.html.markdown

```diff
@@ -0,0 +1,31 @@
+---
+subcategory: "ODS Runbook"
+layout: "nsxt"
+page_title: "NSXT: policy_ods_pre_defined_runbook"
+description: Policy ODS pre-defined runbook data source.
+---
+
+# nsxt_policy_ods_pre_defined_runbook
+
+This data source provides information about policy ODS pre-defined runbook configured on NSX.
+This data source is applicable to NSX Policy Manager and VMC.
```

Comment:
I haven't seen this on VMC - are you sure?

Reply:
No it's a mistake